frame/lending/src/lib.rs

```diff
@@ -427,6 +429,8 @@ pub mod pallet {
 		BorrowRentDoesNotExist,
 
 		MaxLiquidationBatchSizeExceeded,
+
+		VeryOldPrice,
```

Comment:
Perhaps `PriceTooOld`

and maybe have one for collateral and one for borrow? I.e. `BorrowAssetPriceTooOld`, `CollateralAssetPriceTooOld`